linalg/solvers.hpp

```diff
@@ -621,6 +636,7 @@ class MINRESSolver : public IterativeSolver
 
    virtual void SetOperator(const Operator &op);
 
+   /// Iterative solution of the linear system using the Minres method
```

Comment:
pedantic: but for consistency with comment on l. 617 and "GMRES", "Minres" should be "MINRES"